be/test/exec/jni_scanner_test.cpp

```diff
@@ -0,0 +1,317 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "exec/jni_scanner.h"
+
+#include <gtest/gtest.h>
+
+#include "runtime/descriptor_helper.h"
+#include "runtime/runtime_state.h"
+#include "util/thrift_util.h"
+
+namespace starrocks {
+
+struct SlotDesc {
```

Comment:
data structure defined in .cpp file, better surrounded with `namespace {}`, avoid conflicting in other compile unit without even notice.